brave/utility/importer/chrome_importer.cc

```diff
@@ -258,9 +259,18 @@ void ChromeImporter::ImportCookies() {
   std::vector<ImportedCookieEntry> cookies;
   while (s.Step() && !cancelled()) {
     ImportedCookieEntry cookie;
-    base::string16 host(base::UTF8ToUTF16("*"));
-    host.append(s.ColumnString16(0));
-    cookie.domain = s.ColumnString16(0);
+    base::string16 host;
+    base::string16 host_key = s.ColumnString16(0);
+    if (host_key.empty()) {
+      continue;
+    }
+    // Only append * for .domain
+    if (base::StartsWith(base::UTF16ToUTF8(host_key),
+                         ".", base::CompareCase::INSENSITIVE_ASCII)) {
+      host.append(base::UTF8ToUTF16("*"));
```

Comment:
`host_key` is extracted from sql and it will be directly used as `ImportedCookieEntry.domain`
`host` is determined by `host_key` and will be used as `ImportedCookieEntry.host`